test-suites/tests/test_flashloan.rs

```diff
@@ -0,0 +1,55 @@
+#![cfg(test)]
+use pool::{FlashLoan, Request, RequestType };
+use soroban_sdk::{vec, Vec};
+use test_suites::{create_fixture_with_data, moderc3156::create_flashloan_receiver, test_fixture::{TokenIndex, SCALAR_7}};
+
+#[test]
+fn test_liquidations() {
+    let fixture = create_fixture_with_data(false);
+    let frodo = fixture.users.get(0).unwrap();
+    let pool_fixture = &fixture.pools[0];
+    let (receiver_address, receiver_client) = create_flashloan_receiver(&fixture.env, true);
+    receiver_client.init(frodo);
+    
+    // reset frodo's health. 
+    // TODO: there probably is a user in the fixture that doesn't require this due to low hf? haven't really looked into it.
```

Comment:
you can just create a new user, see `wasm_happy_path` for example, we create user `samwise` to start from scratch.

`frodo` mainly exists to seed the test pool with capital.